src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -169,6 +169,7 @@ const UserProfilePicture: React.FC<UserProfilePictureProp> = ({
 UserProfilePicture.defaultProps = {
   avatarClassName: "profile-icon",
   miniAvatars: false,
+  isAudioEffectDisabled: true,
 };
```

Comment:
[NB] It would be nice to refactor this to our standard pattern of providing defaults at the top of the component, rather than this `ComponentName.defaultProps` pattern.